src/components/TodoHeader/TodoHeader.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import React, { FormEvent, useState } from 'react';
+import { Todo } from '../../types/Todo';
+
```

Comment:
There is an unnecessary empty line after importing the Todo type. Please remove the empty line to keep the code clean. 😇